src/main/resources/data/minecolonies/researches/civilian/1_higherlearning.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "higherlearning",
+  "name": "Higher Learning",
+  "branch": "civilian",
+  "exclusiveChildResearch": true,
+  "requiredUniversityLevel": 1,
+  "requiredItems":
+  [
+  {
+    "itemName": "minecraft:book",
+    "itemCount": 3
+  }
+  ],
+  "requiredBuildings":
+  [
+  {
+    "building": "citizen",
+    "buildingLevel": 3
+  }
+  ],
+  "effects":
```

Comment:
I'm wondering if we should have a separate JSON for Research effects, and then have the researches refer to them by ID.

Reply:
It's definitely something I was considering: the systems management side of me _really_ doesn't like all those string constants.  Ideal situation would be to have a pile of hooks set up everywhere in as generic a situation as possible, and then only apply the relevant ones -- I could plausibly see even some behaviors like a modpack where a Quest locks access to (tiers of the!) Builder Hut behind a Research unlocked externally, or where certain citizens can only use certain tools after getting the University to a tier.

  Just a little nervous about digging too deep into the implementation side of things without more planning on the topic.

Reply:
I hear you. This is why I was thinking we should separate the effects now. We can then design what we want the effect definition to look like, and evolve that separately from the research.

Reply:
I see that creating more work for potentially everyone involved. This is currently a pretty satisfactory way of expressing effects, and I think it's outside the bounds of this research system to make it any more complicated. Maybe I just don't quite get what you have in mind, idk.

Reply:
Eh, I find the multi-level effects defined across multiple files problematic. I'd prefer to have a single effect file with levels defined, and then have the researches refer to effect and level.  In some ways I think about the potions system, where the effects are separate from the item that applies them.